pkg/reader/ccip_test.go

```diff
@@ -58,9 +59,6 @@ func TestCCIPChainReader_getSourceChainsConfig(t *testing.T) {
 		}, nil, chainC, offrampAddress,
 	)
 
-	sourceCRs[chainA].On("Bind", mock.Anything, mock.Anything).Return(nil)
-	sourceCRs[chainB].On("Bind", mock.Anything, mock.Anything).Return(nil)
-	destCR.On("Bind", mock.Anything, mock.Anything).Return(nil)
```

Comment:
Technically, `destCR.Bind` was the only one which needed to move (The offramp is now bound during`newCCIPChainReaderInternal`). Moved the rest up for consistency and switched to the `.EXPECT()` for better readability.